users/tracing/users.go

```diff
@@ -13,14 +13,15 @@ import (
 )
 
 const (
-	saveOp            = "save"
-	updateOp          = "update"
-	retrieveByEmailOp = "retrieve_by_email"
-	retrieveByIDOp    = "retrieve_by_id"
-	retrieveByIDsOp   = "retrieve_by_ids"
-	retrieveAllOp     = "retrieve_all"
-	updatePasswordOp  = "update_password"
-	changeStatusOp    = "change_status"
+	save            = "save"
+	update          = "update"
+	retrieveByEmail = "retrieve_by_email"
+	retrieveByID    = "retrieve_by_id"
+	retrieveByIDs   = "retrieve_by_ids"
+	retrieveAll     = "retrieve_all"
+	backupAll       = "backup_all"
+	updatePassword  = "update_password"
+	changeStatus    = "change_status"
```

Comment:
This naming is not consistent. I'm not sure if we can or not know the database name from the tracing wrapper but it would be better to follow the DB naming as it's done here. Can you verify if please?